<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RateServiceChangeView</class>
 <widget class="QWidget" name="RateServiceChangeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>831</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Rates</string>
   </property>
   <widget class="QListWidget" name="lwRates">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbRateInfo">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChangeRate">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Change rate</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>831</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Services</string>
   </property>
   <widget class="QListWidget" name="lwServices">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbServiceInfo">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChangeService">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Add service</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lbResponse">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
